app/src/main/java/gov/hhs/cdc/trustedintermediary/organizations/OrganizationsSettings.java

```diff
@@ -28,23 +29,30 @@ public static OrganizationsSettings getInstance() {
 
     private OrganizationsSettings() {}
 
-    public List<Organization> getOrganizations() {
+    public Map<String, Organization> getOrganizations() {
         return organizations;
     }
 
     public void loadOrganizations(Path filePath) throws IOException, FormatterProcessingException {
         String organizationsFileString = Files.readString(filePath);
-        this.organizations =
+        List<Organization> organizationList =
                 formatter.convertYamlToObject(
                         organizationsFileString, new TypeReference<List<Organization>>() {});
+
+        organizations =
+                organizationList.stream()
+                        .collect(
+                                Collectors.toMap(
+                                        Organization::getName, organization -> organization));
```

Comment:
I think you can do something with `Identity`?  It's been a while.

```suggestion
                                        Organization::getName, Identity::identity));
```

Reply:
@halprin @basiliskus I'm not seeing an import for `Identity`, and the IDE is telling me there is no local variable? Where is `Identity::identity` coming from? 
I added screenshot below for reference, I haven't tried to push anything yet.
![Screenshot 2023-06-01 at 8 12 29 PM](https://github.com/CDCgov/trusted-intermediary/assets/86614374/7b60abaf-7259-4363-ab2a-295c4e79139e)

Reply:
@tjohnson7021, sorry, I mis-remembered.  It's `Function.identity()`.  It's the equivalent to the `something -> something` Java lambda.